examples/llama2/llama2_genai.json

```diff
@@ -28,7 +28,11 @@
         "exporter": {
             "type": "GenAIModelExporter",
             "config": {
-                "precision": "int4"
+                "precision": "int4",

```

Comment:
Do we need a separted pass which set 'metadata_only' as True to prepare the metadata in this example?

Reply:
If we want an example to showcase this feature then yes. I can modify the example and update the script accordingly.

Reply:
Done!